recipes/winpcap/all/test_package/test_package.c

```diff
@@ -0,0 +1,107 @@
+/* Copied and modified from Examples/misc/basic_dump.c */
+
+#include "pcap.h"
+
+#include <stdlib.h>
+#include <time.h>
+
+/* prototype of the packet handler */
+void packet_handler(u_char *param, const struct pcap_pkthdr *header, const u_char *pkt_data);
+
+int main()
```

Comment:
This test package is interactive. It should be made non-interactive.

Reply:
FIXME: borrow test package from https://github.com/bincrafters/community/pull/1395